### PR TITLE
Fixup conmon documentation

### PR DESCRIPTION
**Pull request by haircommander**
there were some leftover unattended to installation guides

Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: install.md [hunk 1]
```diff
@@ -188,13 +188,12 @@ export PATH=$GOPATH/bin:$PATH
...
-To build from source, use the following (if not already executed above, run `export GOPATH=~/go && mkdir -p $GOPATH`):
+To build from source, use the following
 
 ```bash
-git clone https://github.com/cri-o/cri-o $GOPATH/src/github.com/cri-o/cri-o
-cd $GOPATH/src/github.com/cri-o/cri-o
-mkdir bin
-make bin/conmon
+git clone https://github.com/containers/conmon
+cd conmon
```
Comment (TomSweeneyRedHat) [inline install.md]: Don't you still need to define a GOPATH?

File: install.md [hunk 2]
```diff
@@ -188,13 +188,12 @@ export PATH=$GOPATH/bin:$PATH
 #### conmon
 
 The latest version of `conmon` is expected to be installed on the system. Conmon is used to monitor OCI Runtimes.
-To build from source, use the following (if not already executed above, run `export GOPATH=~/go && mkdir -p $GOPATH`):
+To build from source, use the following
```
Comment (TomSweeneyRedHat) [inline install.md]: Please add a colon at the end.

File: install.md [hunk 3]
```diff
@@ -188,13 +188,12 @@ export PATH=$GOPATH/bin:$PATH
...
 
 ```bash
-git clone https://github.com/cri-o/cri-o $GOPATH/src/github.com/cri-o/cri-o
-cd $GOPATH/src/github.com/cri-o/cri-o
-mkdir bin
-make bin/conmon
+git clone https://github.com/containers/conmon
+cd conmon
+make
```
Comment (TomSweeneyRedHat) [inline install.md]: does the make call make the bin dir now, or do you need to do a mkdir?

Comment (haircommander) [reply install.md]: make creates bin/ :)

File: install.md (hunk 1 above)
Comment (haircommander) [reply install.md]: nope conmon doesn't need it!

File: install.md (hunk 2 above)
Comment (haircommander) [reply install.md]: fixed!

**Pull request merged**